spec/models/book_spec.rb

```diff
@@ -0,0 +1,110 @@
+require 'rails_helper'
+
+describe Book do
+  it 'validates presence of genre' do
+    :genre.nil?
+  end
+
+  it 'validates presence of author' do
+    :author.nil?
+  end
+
+  it 'validates presence of image' do
+    :image.nil?
+  end
+
+  it 'validates presence of title' do
+    :title.nil?
+  end
+
+  it 'validates presence of publisher' do
+    :publisher.nil?
+  end
+
+  it 'validates presence of year' do
+    :year.nil?
+  end
+
+  subject(:book) do
+    described_class.new(
+      genre: genre, author: author, image: image, title: title,
+      publisher: publisher, year: year
+    )
+  end
+
+  let(:genre)            { Faker::Book.genre }
```

Comment:
If you aren't going to keep the same indentation between ) and {, just keep them seperated with just one space (in general we prefer this to avoid merge conflicts and differences across diferent team members)